saleor/graphql/product/mutations/products.py

```diff
@@ -393,6 +410,11 @@ def perform_mutation(cls, _root, info, collection_id, products):
             update_products_discounted_prices_of_catalogues_task.delay(
                 product_ids=[pq.pk for pq in products]
             )
+        transaction.on_commit(
+            lambda: [
+                info.context.plugins.product_updated(product) for product in products
```

Comment:
Should we make queryset with prefetched needed data from `products` list?

Reply:
Yes, I will add prefetch.